nginx-auto-config.go

```diff
@@ -30,22 +32,22 @@ var red = color.New(color.FgRed)
 
 func main() {
 	var server service
-	server.verifyRoot = true
 
 	if len(os.Args) > 1 {
-		pArgs := os.Args[1]
-		if pArgs == "-h" || pArgs == "-help" || pArgs == "--help" {
+		pArg := os.Args[1]
+		if pArg == "-h" || pArg == "-help" || pArg == "--help" {
 			fmt.Println("nginx-auto-config is a program which allows you to create configurations for the nginx web server using a number of presets interactively\nLicensed under the MIT license, created by Sidharth Soni (Sid Sun)\nYou can find the source code at: https://github.com/Sid-Sun/nginx-auto-config")
-			os.Exit(0)
-		} else if pArgs == "version" {
-			fmt.Printf("3.2\n")
-			os.Exit(0)
-		} else if pArgs == "-skiproot" || pArgs == "-s" || pArgs == "--skiproot" {
-			server.verifyRoot = false
+		} else if pArg == "--version" || pArg == "-v" || pArg == "-version" {
+			fmt.Println(sVers)
 		} else {
-			fmt.Printf("Unknown option(s) %s, run with -h, -help or --help to get help, -s, -skiproot or --skiproot to skip server root directory validation or without any argumets to launch the program\n", pArgs)
+			fmt.Printf(
+					"Unknown option(s) %s\n" +
```

Comment:
option**, only one argument is taken in the `pArg` var

Reply:
Rather cosmetic change. Better to take note of it and fix it with some other significant commit.

We are still saying "option(s)" not "options" so it will do fine.